facenet/bestmodel.txt

```diff
@@ -0,0 +1 @@
+./checkpoints/1569777463/facenet_3.h5
```

Comment:
please remove the h5 files, can you keep them in GCS and just reference the paths here? 
`facenet/checkpoints/1569777463/facenet_3.h5`

Reply:
also whats the file named as `...kpoints/1569777463/logs/1569777463.1938202/events.out.tfevents.1569777481.ml-instance-dev` can you please delete it?

Reply:
ok

Reply:
deleted